cogs/guildOwners.py

```diff
@@ -393,24 +382,26 @@ async def activate(self, ctx, r: Role):
                 msg_title = ':ballot_box: __Ballot Voting System Error__ :ballot_box: '
                 message = f'System could not activate Ballot Voting pools due to backend issue. Please contact' \
                           f' crypto link team.'
-                await customMessages.system_message(ctx=ctx, sys_msg_title=msg_title, message=message, color_code=0,
+                await customMessages.system_message(interaction=interaction, sys_msg_title=msg_title, message=message,
+                                                    color_code=0,
                                                     destination=1)
         else:
             msg_title = ':ballot_box: __Ballot Voting System Already Active__ :ballot_box: '
-            message = f'You have already activated {ctx.guild} server for voting functionality.'
-            await customMessages.system_message(ctx=ctx, sys_msg_title=msg_title, message=message, color_code=0,
+            message = f'You have already activated {interaction.guild} server for voting functionality.'
+            await customMessages.system_message(interaction=interaction, sys_msg_title=msg_title, message=message,
+                                                color_code=0,
                                                 destination=1)
 
     @activate.error
-    async def activate_error(self, ctx, error):
+    async def activate_error(self, interaction, error):
         if isinstance(error, commands.CheckFailure):
             message = f"You are required to specify active role on the server which will have access to the " \
-                      f"Ballot voting system management. `{self.command_string}owner ballot activate @discord.Role'"
-            await customMessages.system_message(ctx=ctx, color_code=1, message=message, destination=1,
+                      f"Ballot voting system management. `/owner ballot activate @discord.Role'"
+            await customMessages.system_message(interaction=interaction, color_code=1, message=message, destination=1,
                                                 sys_msg_title="Ballot voting access error!")
         elif isinstance(error, commands.MissingRequiredArgument):
             message = f"You forgot to provide @discord.Role which will have access to ballot functions."
-            await customMessages.system_message(ctx=ctx, color_code=1, message=message, destination=1,
+            await customMessages.system_message(interaction=interaction, color_code=1, message=message, destination=1,
```

Comment:
changes are good...maybe you could rewrite the system_message function as destination is perhaps not needed anymore ;)

Reply:
here:
https://github.com/Crypto-Link-Payments/crypto-link/blob/71391fc04ed71e23ebe86f3c8a332a73b3dfd90e/cogs/utils/systemMessaages.py#L136